docs/src/pages/de/core-concepts/component-hydration.md

```diff
@@ -0,0 +1,114 @@
+
+Doch manchmal wird Client-seitiges JavaScript zwingend benötigt. Diese Anleitung zeigt, wie interaktive Komponenten in Astro funktionieren - unter Verwendung einer Technik die partielle Anreicherung (Partial Hydration) genannt wird.
+
+```astro
+---
+// Beispiel: Eine React-Komponente importieren und anschließend verwenden.
+// Standardmäßig rendert Astro dies als HTML und CSS während deines Build
+// - ohne Client-seitiges JavaScript.
+// (Du brauchst Client-seitiges JavaScript? Lies weiter...)
+import MeineReactKomponente from '../components/MeineReactKomponente.jsx';
+---
+<!-- 100% HTML, Null JavaScript! -->
+<MeineReactKomponente />
+```
+
+## Konzept: Partielle Anreicherung
+
+Es gibt viele Fälle in denen du eine interaktive UI-Komponente benötigst, die im Browser ausgeführt wird:
+
+- Ein Bilder-Karussel
+- Eine Sucheingabe mit Auto-Vervollständigung
+- Ein Toggle-Button zum Öffnen und Schließen einer Sidebar
+- Ein "Jetzt kaufen"-Button
+
+In Astro ist es deine Aufgabe als Entwicklerin oder Entwickler innerhalb einer Seite die Ausführbarkeit von Komponenten im Browser ausdrücklich zu "deklarieren". Astro kann anschließend diese Information verwenden, um herauszufinden welches JavaScript benötigt wird - und die Seite um exakt das anreichern, was benötigt wird. Diese Technik wird als Partielle Anreicherung (Partial Hydration) bezeichnet.
+
+**Partielle Anreicherung** -- das ausschließliche Anreichern derjenigen Komponenten, die JavaScript benötigen, während der Rest der Seite statisches HTML bleibt -- klingt eigentlich sehr einfach - und das sollte es auch sein! Websites wurden über Jahrzehnte so gebaut. Und es kam erst in den letzten Jahren dahin, dass so genannte Single-Page-Anwendungen (SPAs) die Idee ins Spiel brachten, dass deine gesamte Website in JavaScript geschrieben werden könne, und das Ganze schließlich von jeder einzelnen Nutzerin und jedem einzelnen Nutzer im Browser kompiliert und ausgeführt werden könne.
+
+_Beachte: Partielle Anreicherung (Partial Hydration) wird manchmal auch als "schrittweise Erweiterung" or "schrittweise Anreicherung" bezeichnet. Während durchaus leichte Unterschiede zwischen den Begriffen bestehen, kannst du sie für unsere Zwecke als verschiedene Synonyme des selben Konzeptes betrachten._
+
+**Partielle Anreicherung ist das Geheimnis hinter der Geschichte von der standardmäßigen Schnelligkeit von Astro.** Next.js, Gatsby und andere JavaScript-Bibliotheken können partielle Anreicherung nicht unterstützen, da sie deine gesamte Website als eine einzelne JavaScript-Anwendung betrachten.
+
+## Konzept: Architektur aus Inseln
+
+**Architektur aus Inseln** ist die Idee partielle Anreicherung zu verwenden, um komplette Websites zu erzeugen. Architektur aus Inseln ist eine Alternative zu der beliebten Idee deine gesamte Website in eine Client-seitige JavaScript-Anwendung zu packen, welche von den Nutzerinnen und Nutzern heruntergeladen werden muss.
```

Comment:
We got the problem here that it is called 'Island_s_ Architecture"  for a reason (it is an architecture consisting of islands) so we can't omit the plural form without perverting or neglecting the idea. Since it doesn't sound right do translate it in a plural form into German we should stick with the original here as well I guess.